### ISSUE TITLE
Reproducible issue

### ISSUE THREAD

**Issue by HuanjunRan** [sanvihs-05/RAKSHAK-Micro-Doppler-based-target-classification-#1]
I want to reproduce your work, but  my result is vary bad that accuracy is only 0.5x.
I choose tensorflow 2.7.0 and CUDA 11.2 ,and 4GB RTX3050.
Especially ,the training of the autoencoder is good(loss is low 0.003),but the loss of  classification is up 2.x.
I dont know why,maybe GPU is small?
Hope to get your answer ! Thanks.

**Comment by Liang01-81** [comment]
Hello, I am very interested in your research. May I ask if there are any papers you could recommend related to your work? I would greatly appreciate any references you could share. I look forward to your reply.Thanks
